packages/infra/lib/api-stack.ts

```diff
@@ -282,87 +282,6 @@ export class APIStack extends Stack {
       uri: `http://${apiLoadBalancerAddress}/{proxy}`,
     });
 
-    //-------------------------------------------
```

Comment:
I moved this below api gateway so that the cw lambda was able to be granted read and write access

Reply:
Makes sense, i updated the file